src/runHandler.ts

```diff
@@ -76,6 +76,8 @@ export async function runHandler(
 
   run.end()
   log.info('Tests run end')
+
+  await vscode.commands.executeCommand('editor.action.refreshCodeLens');
```

Comment:
We will refresh all the code lens here after the test so ensure they have up to date coverage information